physics/speeds_of_gas_molecules.py

```diff
@@ -1,46 +1,39 @@
-"""
+r"""
 The root-mean-square, average and most probable speeds of gas molecules are
 derived from the Maxwell-Boltzmann distribution. The Maxwell-Boltzmann
 distribution is a probability distribution that describes the distribution of
 speeds of particles in an ideal gas.
 
 The distribution is given by the following equation:
-
-        -------------------------------------------------
-        | f(v) = (M/2πRT)^(3/2) * 4πv^2 * e^(-Mv^2/2RT) |
-        -------------------------------------------------
+    .. math:: f(v) = \left(\frac{M}{2 \pi RT}\right)^{\frac{3}{2}} \cdot 4 \pi v^2
+                     \cdot e^{-\frac{Mv^2}{2RT}}
```

Comment:
The current text is readable by human code readers, while the second forces the reader to open the docs to understand the formula. Would it be possible/useful to keep both the new and the old?

Reply:
Done